doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -13,7 +13,12 @@ The following construction should be treated as an informed hypothesis rather th
 
 ## Enumerating Stakeholders
 
-Stakeholders in vulnerability management can be identified within multiple independent axes. For example, they can be identified by their responsibility: whether the organization *supplies*, *deploys*, or *coordinates* remediation actions. Organizations may also be distinguished by type of industry sector. While it might be useful to enumerate all the sectors of the economy, we propose to draft decision points that include those from multiple important sectors. For example, we have safety-related questions in the decision path for all suppliers and deployers, so whether or not the stakeholder is in a safety-critical sector, the decision will be addressed.
+Stakeholders in vulnerability management can be identified within multiple independent axes. 
+For example, they can be identified by their responsibility: whether the group *supplies*, *deploys*, or *coordinates* remediation actions. 
+Depending what task a team is performing in a supply chain, the team may be considered a supplier, deployer, or a coordinator.
+Therefore, one organization may have teams that take on different roles.
+For example, an organization may have a development team may take on the supplier stakeholder role for the software it produces and an internal IT team that takes on a deployer stakeholder role for, as an example, managing the email clients its employees use. 
+Organizations may also be distinguished by type of industry sector. While it might be useful to enumerate all the sectors of the economy, we propose to draft decision points that include those from multiple important sectors. For example, we have safety-related questions in the decision path for all suppliers and deployers, so whether or not the stakeholder is in a safety-critical sector, the decision will be addressed.
 
```

Comment:
For example, a large organization may have a development team which may take on the supplier stakeholder role for the software it produces and an internal IT team that takes on a deployer stakeholder role for, as an example, managing the email clients its employees use. In a small organization the stakeholder role may exist within one group. In this example, a middle-ware software company receives a patch notification from their supplier and the stakeholder decides to incorporate it into their next update deployment -- instead of pushing an immediate mitigation.

[Does this make sense to you??]

Reply:
I'm not sure that the "small org" comment is so readily connected to the libraries / supply chain issue. 
This first part could be cleanly stated:
> Especially in smaller organizations, a team may act in different stakeholder roles at different times.
after "employees use."

Library use is ubiquitous, so I don't think linking this to org size is helpful. I think this is a separate thought that goes somewhere else? 
> A supplier who uses a third-party library in their software may get a patch from the library supplier, and thus need to make a decision as a deployer about applying the library's patch.
But this this sort of library situation seems like it's complicated, I think we should talk about that somewhere else. And it's probably a separate issue than updating the section headers / intros?

Reply:
Having "for...example" show up within an example is making this a bit more complex than it needs to be. Also, I don't think that large/small is the right axis to discriminate on - I think it's more of a devops/not-devops axis.

Here's my suggested replacement:

For example, an organization that develops and uses its own software might delegate the supplier role to its development team and the deployer role to its IT operations team. 
On the other hand, organizations using a DevOps approach to providing services might have a single group responsible for both the supplier and deployer roles.

Reply:
I like it!

Reply:
I think that works.